master/buildbot/test/unit/test_db_migrate_versions_044_add_step_hidden.py

```diff
@@ -40,7 +40,7 @@ def create_tables_thd(self, conn):
             'steps', metadata,
             sa.Column('id', sa.Integer, primary_key=True),
             sa.Column('number', sa.Integer, nullable=False),
-            sa.Column('name', sa.String(50), nullable=False),
+            sa.Column('name', sa.String(70), nullable=False),
```

Comment:
You will have to create a new migration script in order to increase this restriction.
I believe this was a bad idea to limit the size of name inside the db. probably it will be better to change the column to sa.Text() so that there is no limitation.

Reply:
I know this pull is failing and I'm with you -- no restriction would be better. Can we put the desire to change the step length to a Text field instead on a list somewhere? What would it take to simply make that change? We would need some kind of upgrade process?

Reply:
@sphet not sure I fully get your comment. I updated my review in order to take in account what I understand.

Reply:
@tardyp All I meant was 1) please no restrictions - let's move to a variable-length text field 2) Can we ensure this change is on the roadmap somewhere. Thanks!

Reply:
@tardyp @sphet Agree. However, I am not much familiar with database migration script,  and I am afraid that I wouldn't have much time to learn about it and implement the changes.

Would anyone of you have time to implement this?